pyaedt/modules/report_templates.py

```diff
@@ -1,8 +1,148 @@
+        self._oreport_setup.ChangeProperty(
+            ["NAME:AllTabs", ["NAME:Limit Line", ["NAME:PropServers", self.line_name], props_value]]
+        )
+        return True
+
+    @pyaedt_function_handler()
+    def set_line_properties(
+        self, style=None, width=None, hatch_above=None, violation_emphasis=None, hatch_pixels=None, color=None
+    ):
+        """Set Trace Properties.
+
+        Parameters
+        ----------
+        style : str or
+            Limit Line Style. LINESTYLE PROPERTY can be used
+        width : int
+            Limit Line width.
+        hatch_above : bool
+            Either if Hatch above the limit line or not.
+        violation_emphasis : bool
+            Either if to add violation emphasis or not.
+        hatch_pixels : int
+            Hatch pixel numbers.
+        color : tuple or list
+            Trace Color. a Tuple of (R,G,B) with int of [0,255] has to be provided.
+
+        Returns
+        -------
+        bool
+        """
+        props = ["NAME:ChangedProps"]
+        if style:
+            props.append(["NAME:Line Style", "Value:=", style])
+        if width and isinstance(width, (int, float)):
+            props.append(["NAME:Line Width", "Value:=", str(width)])
+        if hatch_above and isinstance(hatch_pixels, int):
+            props.append(["NAME:Hatch Above", "Value:=", hatch_above])
+        if hatch_pixels and isinstance(hatch_pixels, int):
+            props.append(["NAME:Hatch Pixels", "Value:=", str(hatch_pixels)])
+        if violation_emphasis:
+            props.append(["NAME:Violation Emphasis", "Value:=", violation_emphasis])
+        if color and isinstance(color, (list, tuple)) and len(color) == 3:
+            props.append(["NAME:Color", "R:=", color[0], "G:=", color[1], "B:=", color[2]])
+        return self._change_property(props)
+
+
+class Trace(object):
+    """Trace Management Class."""
+
+    def __init__(self, report_setup, trace_name):
+        self._oreport_setup = report_setup
+        self.trace_name = trace_name
+        self.LINESTYLE = LineStyle()
+        self.TRACETYPE = TraceType()
+        self.SYMBOLSTYLE = SymbolStyle()
+
+    @pyaedt_function_handler()
+    def _change_property(self, props_value):
+        self._oreport_setup.ChangeProperty(
+            ["NAME:AllTabs", ["NAME:Attributes", ["NAME:PropServers", self.trace_name], props_value]]
+        )
+        return True
+
+    @pyaedt_function_handler()
+    def set_trace_properties(self, trace_style=None, width=None, trace_type=None, color=None):
+        """Set Trace Properties.
+
+        Parameters
+        ----------
+        trace_style : str
+            Limit Line Style. LINESTYLE property can be used.
+        width : int
+            Trace Width.
+        trace_type : str
+           Trace Style. TRACETYPE property can be used.
+        color : tuple or list
+            Trace Color. a Tuple of (R,G,B) with int of [0,255] has to be provided.
```

Comment:
```suggestion
       trace_style : str, optional
            Style for the trace line. The default is ``None``. You can also use
            the ``linestyle`` property.
        width : int, optional
            Width of the trace line. The default is ``None``.
        trace_type : str
           Type of the trace. The default is ``None``. You can also use the ``tracetype`` property.
        color : tuple, list
            Trace color. A tuple of (R,G,B) or a list of integers [0,255] must be
            provided. The default is ``None``.
```

Reply:
Same comments here as for the previous, similar properties. I'm also not sure why the names differ.